proto/axelar/axelarnet/v1beta1/events.proto

```diff
@@ -74,12 +74,16 @@ message FeeCollected {
 }
 
 message ContractCallSubmitted {
-  string sender = 1;
-  string destination_chain = 2
+  bytes sender = 1 [ (gogoproto.casttype) =
```

Comment:
this does need to be a string if we reuse the event for cosmos chain gmp calls @haiyizxx